tests/system/python/service/test_service_notification.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test_service_notification with:
+        Service Notification And OverMaxRule in built rule plugin
+        notify-python35 delivery plugin
+"""
+
+import os
+import time
+import subprocess
+import http.client
+import json
+import pytest
+
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+SERVICE = "notification"
+SERVICE_NAME = "NotificationServer #1"
+NOTIFY_PLUGIN = "python35"
+NOTIFY_RULE = "OverMaxRule"
+
+
+class TestServiceNotification:
+
+    def _configure_and_start_service(self, service_branch, foglamp_url, remove_directories):
+        try:
+            subprocess.run(["$FOGLAMP_ROOT/tests/system/python/scripts/install_c_service {} {}"
+                           .format(service_branch, SERVICE)], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "{} installation failed".format(SERVICE)
+
+        # Start service
+        conn = http.client.HTTPConnection(foglamp_url)
+        data = {"name": SERVICE_NAME,
+                "type": "notification",
+                "enabled": "true"
+                }
+        conn.request("POST", '/foglamp/service', json.dumps(data))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert 2 == len(jdoc)
+        assert SERVICE_NAME == jdoc['name']
+
+        remove_directories("/tmp/foglamp-service-{}".format(SERVICE))
+
+    def _install_notify_plugin(self, notify_branch):
```

Comment:
NOTIFY_PLUGIN should not be used here from defined const, but by the caller itself. here we need that as arg. so that same def can be utilized for future tests addition